project/ws/app/src/lib/routes/network-v2/routes/network-connection-requests/network-connection-requests.component.html

```diff
@@ -39,16 +39,18 @@ <h4 class="w-full text-center ws-mat-black60-text" *ngIf="data && data.length <=
                         <ws-app-careers-card class="w-fix" [discuss]="discuss"></ws-app-careers-card>
                     </div>
                 </ng-container> -->
-                <ng-container *ngIf="data && data.length">
+                <ng-container *ngIf="filterdData && filterdData.length">
```

Comment:
same as above

Reply:
**filterdData** is alreay defined as array. I think no need to do optional check again.